### PR TITLE
Fix christmas bug

### PR DESCRIPTION
**Pull request by Privat33r-dev**
Birds count has been changed from 1000 to 10.
Random pipe speed multiplier variable has been changed to constant value (1).

### CODE REVIEW

**Comment by nice6599** [comment]
Very helpful.

File: flappyBird/sketch.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ function setup() {
 
   pauseBecauseDead = false;
 
-  population = new Population(1000);
+  population = new Population(10);
```
Comment (Prometheos2) [inline flappyBird/sketch.js:65]: Why did you change that value, if I may ask?

Comment (Privat33r-dev) [reply flappyBird/sketch.js:65]: I changed this value because too big population can cause lags. Of course you can change population to 50 or 100 and it will be even better.

**Comment by Privat33r-dev** [comment]
PR #8 provides better solution and description.
